open-labz/react-javascript-starter
# Add new React components

#### What does this PR do?
In this PR, we create React components inside the */src* directory, using Atomic Design principles to organise our component file structure.

We use the following toolbox: 
- [Atomic Design](https://atomicdesign.bradfrost.com/chapter-2)
- [React Router](https://www.npmjs.com/package/react-router-dom)

#### Description of Task to be completed?
- Install Router to switch between components
- Set up nested components with props.
- Create parent layouts to hold different views

#### How should this be manually tested?

> Once you have cloned and setup the project repo locally, and installed dependencies with `npm install`...
 
- In your terminal, browse to the project directory.
- Run the npm script `npm run start:dev` to trigger the build process.
- Visit the localhost URL in the browser.
- You should be able to see a landing page for the app.

#### Any background context you want to provide?
- Try to refer to the [React official docs](https://reactjs.org/docs/getting-started.html).

#### What are the relevant user stories?
N/A

#### Screenshots

##### App Landing Page
![Screenshot: App landing page](https://user-images.githubusercontent.com/65714243/97009419-5d2b2480-1544-11eb-8ca5-fddd6ca2d16f.png)
